src/main/java/codeu/controller/ActivityServlet.java

```diff
@@ -70,15 +71,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 	   /* Get all conversations. */
 	   List<Conversation> conversations = conversationStore.getAllConversations();
 
-	   /* Make the conversations variable accesible to the jsp file. */           
+	   /*Get all users. */
+	   List <User> users = userStore.getAllUsers();
```

Comment:
nit:
Note the spacing

/* Get all users. */
List<User> users = userStore.getAllUsers();